### DISCUSSION TITLE
Trips-Viz appearance

### DISCUSSION THREAD

**Discussion by skiniry** [skiniry/Trips-Viz#1] [Ideas]
"Beautiful", "Pretty", "Attractive" these are just some of the words that have never been used to describe Trips-Viz. It needs a serious makeover to lose it's reputation as the ugly duckling of Riboseq.org tools.

**Comment by JackCurragh** [comment]
First of all, congrats @skiniry that Trips has been called "Prettier" by Pasha in yesterdays literature seminar. 

I just wanted to point out this set of websites which are next level in terms of prettiness [https://iobio.io/](url) from Marth lab. Even though adopting iobio itself would probably be way too much work it gives nice ideas for design
